ref/prometheus.md

```diff
@@ -2,87 +2,206 @@ Anbox Cloud gathers various performance metrics that you can access through API
-| `ams_cluster_container_boot_time_seconds_sum`    | Sum of all container boot times. Can be used to compute the average boot time                                                                                                           | 1.0.0           |
-| `ams_cluster_container_boot_time_seconds_bucket` | Container boot times bucket. Can be used for alerting when above a threshold. See the [Prometheus documentation](https://prometheus.io/docs/practices/histograms/) for more information | 1.0.0           |
-| `ams_cluster_containers_per_application_total`   | Number of containers per application                                                                                                                                                    | 1.0.0           |
-| `ams_cluster_containers_per_status_total`        | Number of containers per container status                                                                                                                                               | 1.0.0           |
-| `ams_cluster_available_cpu_total`                | Total CPUs available in each worker node                                                                                                                                                | 1.0.0           |
-| `ams_cluster_used_cpu_total`                     | Used CPUs in each worker node                                                                                                                                                           | 1.0.0           |
-| `ams_cluster_available_memory_total`             | Total memory available in each worker node                                                                                                                                              | 1.0.0           |
-| `ams_cluster_used_memory_total`                  | Used memory in each worker node                                                                                                                                                         | 1.0.0           |
+    http://<AMS_server>:<AMS_port>/internal/1.0/metrics
+
+Replace `<AMS_server>` with the IP address of your AMS server, which you can determine by running one of the following commands:
+
+* For a full Anbox Cloud deployment: `juju run --unit ams/0 -- unit-get private-address`
+* For the Anbox Cloud Appliance: `juju run -m appliance:anbox-cloud --unit ams/0 -- unit-get private-address`
+
+Replace `<AMS_port>` with the port for the API endpoint, which you can determine by running one of the following commands:
+
+* For a full Anbox Cloud deployment: `juju config ams prometheus_target_port`
+* For the Anbox Cloud Appliance: `juju config -m appliance:anbox-cloud ams prometheus_target_port`
+
+You can then access the endpoint with `curl`, for example:
```

Comment:
We should clarify that this is only possible from a machine on the same network.